pom.xml

```diff
@@ -295,6 +296,13 @@
       <artifactId>maven-invoker</artifactId>
       <version>3.0.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.benf</groupId>
+      <artifactId>cfr</artifactId>
+      <version>${version.cfr}</version>
+      <scope>system</scope>
```

Comment:
Scope `system` will not package the dependency into the resulting jar. You need to create a local maven repository and add it to git. See: https://sookocheff.com/post/java/local-maven-repository/

Reply:
Thanks. I'll do that.